.env

```
@@ -17,7 +17,7 @@
 APP_ENV=dev
 
 #TRUSTED_PROXIES=127.0.0.0/8,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16
-#TRUSTED_HOSTS='^(localhost|example\.com)$'
+TRUSTED_HOSTS='^.*\.dev\.local$'
```

What do you think about checking for `*.projects.w-vision.ch` as well?
```suggestion
TRUSTED_HOSTS='(.*\.dev\.local|.*\.projects\.w-vision\.ch)$'
```

---

@aarongerig changed it to
`TRUSTED_HOSTS='^.*(\.dev\.local|\.projects\.w-vision\.ch)$'`
and tested it.
Fine for you?

---

@alexloetscher95 Yes, even better than my suggestion! You can go ahead and merge it.